app/ui/src/app/integration/edit-page/step-configure/step-configure.component.html

```diff
@@ -64,11 +64,8 @@
           <div *ngSwitchCase="'mapper'">
             <div class="row">
               <div class="col-md-12"
-                   *ngIf="step && inputDataShape && outputDataShape">
-                <syndesis-data-mapper-host [inputDataShape]="inputDataShape"
-                                           [outputDataShape]="outputDataShape"
-                                           [step]="step"
-                                           [position]="position"></syndesis-data-mapper-host>
+                   *ngIf="step">
+                <syndesis-data-mapper-host [step]="step" [position]="position"></syndesis-data-mapper-host>
```

Comment:
probably don't even need to pass the step along TBH, you could get it from the injected `CurrentFlow` service via the position.